### PR TITLE
refactor(style): add standard animation values with example usage

### PR DESCRIPTION
**Pull request by booc0mtaco**
updated `transform` animations in this PR

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @booc0mtaco has signed the CLA. Thanks for the pull request!

File: src/styles/constants/_animation.scss [hunk 1]
```diff
@@ -1 +1 @@
-$bdl-transition-duration-200ms: .2s;
+$bdl-transition-duration-base: .15s;
```
Comment (priyajeet) [inline src/styles/constants/_animation.scss]: Since we are renaming this, might as well convert to camelcase SUIT style, eg: is in the CSS conventions doc

Comment (booc0mtaco) [reply src/styles/constants/_animation.scss]: `$bdl-transitionDurationBase`?

File: src/styles/mixins/_animation.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@mixin bdl-transition($time-length, $properties...) {
```
Comment (priyajeet) [inline src/styles/mixins/_animation.scss]: should time-length be defaulted to the base value? Though then you would have to change args order.

**Pull request merged**